members/SCOL39_250362.yaml

```diff
@@ -0,0 +1,116 @@
+bioguide: SCOL39_250362
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "http://www.tfaforms.com/426053"
+    - find:
+        - selector: input#tfa_2745
+    - click_on:
+        - selector: input#tfa_2745
+    - find:
+        - selector: form#tfa_0 input#tfa_2748
```

Comment:
Do you need the form id in these selectors? Seems like the `#tfa_2748` alone should be enough